src/sentry/static/sentry/app/components/events/interfaces/frame.jsx

```diff
@@ -25,6 +25,44 @@ export function trimPackage(pkg) {
   return filename.replace(/\.(dylib|so|a|dll|exe)$/, '');
 }
 
+class FunctionName extends React.Component {
+  static propTypes = {
+    frame: PropTypes.object,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      fullFunction: false,
+    };
+  }
+
+  toggle = event => {
+    event.stopPropagation();
+    this.setState({
+      fullFunction: !this.state.fullFunction,
```

Comment:
Let's use setState callback when referencing prev state. See: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-access-state-in-setstate.md